javascript/javascript3/week3/circle.html

```diff
@@ -4,8 +4,12 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+
 </head>
 <body>
-   <script src="ownPlaylist.js"></script> 
+    <canvas id="myCanvas" width="1500" height="1500" style="border:1px solid #000000;">
```

Comment:
It is not recommended to style inline and much better to create a style file and using class for styling